sky/global_user_state.py

```diff
@@ -171,11 +226,21 @@ def update_last_use(cluster_name: str):
 
 def remove_cluster(cluster_name: str, terminate: bool):
     """Removes cluster_name mapping."""
+    handle = get_handle_from_cluster_name(cluster_name)
```

Comment:
not used?

Reply:
handle is passed down to the non-terminate case later in the method, so we still need it

Reply:
Ahh, I missed that! Then how about we keep it close to the place we use it to have a better readability?